AlphaWallet/Tokens/Helpers/CallSmartContractFunction.swift

```diff
@@ -141,4 +141,8 @@ func getEventLogs(
     }
 
     return contractInstance.getIndexedEventsPromise(eventName: eventName, filter: filter)
+        .recover { error -> Promise<[EventParserResultProtocol]> in
+            debugLog("[eth_getLogs] failure for server: \(server) with error: \(error)")
```

Comment:
@oa-s Have you seen why/when it happens?

Reply:
in my case i got this when received an error like `PromiseKit:cauterized-error: nodeError("exceed maximum block range: 5000")`, so this pr, help identify server where an error occured

Reply:
Ah, like the one for mumbai? Great!

Reply:
yes right, i also cached similar with some other network.